src/components/InstancesList/InstancesList.js

```diff
@@ -182,68 +185,80 @@ class InstancesView extends React.Component {
     });
   };
 
+  generateCQLQueryReport = async () => {
+    const { data } = this.props;
+
+    const cqlQuery = buildQuery(data.query, {}, data, { log: noop }, this.props);
+
+    exportStringToCSV(cqlQuery);
+  }
+
+  getActionItem = (id, icon, messageId, onClickHandler, isDisabled = false) => {
```

Comment:
I suggest making the params as one object since now there a lot of those.
```suggestion
  getActionItem = ({ id, icon, messageId, onClickHandler, isDisabled = false }) => {
```